README.md

```diff
@@ -18,6 +18,29 @@ case class Person(name: String, age: Int, job: Option[String])
 val person: Person = r.as[Person]
 val person_?: Option[Person] = r.getAs[Person]
 ```
+#### Case Class Type Mapping
+
+| Cassandra Type |     Scala/Java Type    |
+|:--------------:|:----------------------:|
+| varchar        | String                 |
+| uuid           | java.util.UUID         |
+| inet           | java.net.InetAddress   |
+| int            | Int                    |
+| bigint         | Long                   |
+| boolean        | Boolean                |
+| double         | Double                 |
+| varint         | BigInt                 |
+| decimal        | BigDecimal             |
+| float          | Float                  |
+| timestamp      | org.joda.time.DateTime |
+| blob           | java.nio.ByteBuffer    |
+
+Also maps the following types to their equivalent in Cassandra:
+* List
+* Set
+* Map
```

Comment:
also maps `BigInt` and `BigDecimal` to their scala counterparts (of the same name) and `java.nio.ByteBuffer` -> `Array[Byte]`, none of which would happen automatically, usually